src/main/java/com/minecolonies/coremod/entity/ai/mobs/util/MobEventsUtils.java

```diff
@@ -55,31 +53,43 @@ private MobEventsUtils()
 
     public static void barbarianEvent(final World world, final Colony colony)
     {
-        if(world == null)
+        if(world == null || !colony.isCanHaveBarbEvents())
         {
             return;
         }
 
+        System.out.println("debug1");
+
         numberOfSpawns(colony);
 
+        System.out.println("debug2");
+
         final BlockPos targetSpawnPoint = calculateSpawnLocation(world, colony);
 
+        System.out.println("debug3");
+
         if(targetSpawnPoint.equals(colony.getCenter()))
         {
             return;
         }
 
+        System.out.println("debug4");
+
         if (Configurations.gameplay.enableInDevelopmentFeatures)
         {
             LanguageHandler.sendPlayersMessage(
               colony.getMessageEntityPlayers(),
               "Horde Spawn Point: " + targetSpawnPoint);
         }
 
+        System.out.println("debug6");
+
         LanguageHandler.sendPlayersMessage(
           colony.getMessageEntityPlayers(),
           "event.minecolonies.raidMessage");
 
+        System.out.println("debug7");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS106)